src/components/Splash.vue

```diff
@@ -0,0 +1,154 @@
+            <BasicIcon name="math" :iconClass="metadataIconClass"></BasicIcon>
+            <div class="flex items-center">
+              <p :class="metadataTitleClass">{{ subject }}</p>
+            </div>
+          </div>
+          <div :class="metadataCellClass">
+            <BasicIcon
+              name="notepad"
+              :iconClass="metadataIconClass"
+            ></BasicIcon>
+            <div class="flex items-center">
+              <p :class="metadataTitleClass">{{ examType }}</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- start button -->
+      <icon-button
+        :titleConfig="startButtonTextConfig"
+        buttonClass="bg-white hover:bg-gray-200 rounded-lg h-14 w-40  ring-primary px-2 border-b-outset border-primary"
+        class="rounded-2xl shadow-lg mt-4 place-self-center"
+        data-test="startQuiz"
+        @click="start"
+      ></icon-button>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import IconButton from "./UI/Buttons/IconButton.vue";
+import BasicIcon from "./UI/Icons/BasicIcon.vue";
+import { defineComponent, computed, reactive, toRefs } from "vue";
+export default defineComponent({
+  name: "Splash",
+  components: {
+    IconButton,
+    BasicIcon,
+  },
+  props: {
+    title: {
+      type: String,
+      required: true,
+    },
+    subject: {
+      type: String,
+      required: true,
+    },
+    examType: {
+      type: String,
+      required: true,
+    },
+    numQuestions: {
+      type: Number,
+      required: true,
+    },
+    classNumber: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props, context) {
+    const state = reactive({
+      startButtonTextConfig: {
+        value: "Let's Start",
+        class: "text-lg md:text-xl text-primary font-poppins-bold",
+      },
+      metadataContainerClass:
+        "grid grid-cols-2 space-x-6 bg-yellow-400 p-4 rounded-2xl w-11/12 bp-360:w-10/12 sm:w-2/3 md:w-1/2 xl:w-1/3",
+      metadataCellClass:
+        "flex space-x-1 bp-360:space-x-2 sm:space-x-4 border-primary flex space-x-2 sm:space-x-4 border-primary",
+      metadataIconClass: "h-full w-8 sm:w-12 fill-primary flex",
+      metadataTitleClass: "font-poppins-regular text-sm sm:text-base",
+    });
+
+    /** title of the plio. "Untitled" if no title is present */
+    const displayTitle = computed(() => {
+      if (props.title != undefined) return props.title || "Untitled";
+      return "Untitled";
+    });
+
+    function preventScrolling(event: Event) {
+      event.preventDefault();
+    }
+
+    function start() {
+      context.emit("start");
+    }
+
+    return {
+      ...toRefs(state),
+      displayTitle,
+      preventScrolling,
+      start,
+    };
+  },
+  emits: ["start"],
+});
+</script>
+
+<style>
+@import url("https://fonts.googleapis.com/css2?family=Londrina+Solid&family=Poppins:wght@400;700&display=swap");
+
+.font-londrina {
+  font-family: "Londrina Solid";
```

Comment:
`Londrina` and `Poppins` -- where does this font choice come from? Are they light in terms of kb, etc?

Reply:
this was based on the designs that Karn gave me

Reply:
Asking about size because using non-standard Google fonts means a good chance that every client will download it fresh. If it's a standard font, or at least a google font that's likely to exist on their device already (like Roboto), the browser won't download it again.